### DISCUSSION TITLE
tutaj gadu gadu ok please

### DISCUSSION THREAD

**Discussion by xxcriticxx** [MajkiIT/polish-ads-filter#17138] [General] [answered]
gadu gadu

**Comment by krystian3w** [comment, marked as answer]
https://www.ggapp.com/

**Comment by hawkeye116477** [comment]
@xxcriticxx Czy kurier Hotmail/Outlook dostarczył :gift: , którego nadawcą jest IndieGala?

**Reply by hawkeye116477** [reply]
Nie martw się @krystian3w, to nie dla ciebie, ale jak chcesz, to kolejnym razem może wyślę, ale niestety nie posiadam adresu, bo ten z domeną Interia chyba nieaktualny... .
Jak już tak jesteś ciekawy, to chodziło mi o to, że wysłałem :gift: dla @xxcriticxx na jego maila Hotmail za pomocą serwisu IndieGala.

**Reply by krystian3w** [reply]
"Pw" jakoś dociera na adres wycieku morele by dowcipniś z anonimowym mailingiem "Shark Lasers".

![obraz](https://i.imgur.com/m2PI071.png)

**Comment by xxcriticxx** [comment]
@hawkeye116477 hotmail to moja junk poczta ktorej nie sprawdzam

**Reply by hawkeye116477** [reply]
No to żałuj, bo tam czeka dla ciebie :gift: o wartości ok. 36 PLN :smile_cat:

**Comment by xxcriticxx** [comment]
@hawkeye116477 trzeba bylo zatrzymac kase na pizze. dzieki.

**Reply by hawkeye116477** [reply]
Tak naprawdę dostałem to w nagrodę za darmo, a podana wartość jest ze Steama.

PS: Jak odpowiadasz na czyjąś wiadomość, to lepiej pisać w polu „write a reply” niż „suggest an answer".

**Reply by xxcriticxx** [reply]
nie gram w gierki ale dzieki za prezent

**Reply by krystian3w** [reply]
W sumie ciekawe czy ma datę przydatności "zużyć przed" (inną niż bankructwo IndieGala).

**Reply by hawkeye116477** [reply]
Nie ma daty, takie rzeczy to chyba tylko na GOGu.

> Your keys and your account will NEVER expire. 

PS: Dla ciekawskich, ten klucz jest do gry Nuclear Dawn.

**Reply by krystian3w** [reply]
Epic daje "~3 tygodnie" na kod świąteczny wręczany z urzędu, ale pewnie nie organizuje konkursów na inne.

**Comment by hawkeye116477** [comment]
@MajkiIT Czy kurier IndieGala dotarł z :gift:?

**Reply by hawkeye116477** [reply]
To jak tam, czy Szalone Maszyny dojechały do @MajkiIT?

**Comment by hawkeye116477** [comment]
A jak tam z prezentem @krystian3w, czy jest już na wirtualnej akcji :police_car: ?

**Reply by krystian3w** [reply]
Jeszcze nie: 

![obraz](https://user-images.githubusercontent.com/35370833/152687510-b89f2768-16ec-4eac-af2d-afbd1e845b39.png)


Niby ten ocr poprawnie odczytał klucz z png https://www.onlineocr.net/pl/

**Reply by hawkeye116477** [reply]
PS: Tym razem klucz był z Twitcha za „oglądanie" jak grają (3000 pkt, losowy klucz).

**Comment by hawkeye116477** [comment]
Dzisiaj w wyzwaniu IG wygrałem Killing Floor Bundle - June 2013. Po wpisaniu klucza zauważyłem, że w ekwipunku dodatkowo pojawiły się jakieś „przepustki dla gości”. Jak się okazuje podobno pozwalają one na 3 dni darmowego grania w [Killing Floor](https://store.steampowered.com/app/1250/Killing_Floor/) wraz z modem [Defence Alliance 2](https://store.steampowered.com/app/35420/Defence_Alliance_2/) od daty aktywacji.

Tak więc czy ktoś jest zainteresowany takim :gift: , może @krystian3w?

**Reply by krystian3w** [reply]
Nie.

**Reply by xxcriticxx** [reply]
waste of time!

**Comment by hawkeye116477** [comment]
@xxcriticxx mówił, że nie gra w gry, a tymczasem znalazłem jego profil na Steamie => `https://steamcommunity.com/profiles/76561198046772847`. Wbił nawet wyższy poziom niż ja i więcej „lat służby”, ale za to gier ma mniej.

**Reply by hawkeye116477** [reply]
A tak a propos Steama, znalazłem tam też profil @MajkiIT i już wiem dlaczego ciągle robi sobie takie długie kilkunastodniowe przerwy. Otóż okazuje się, że on najprawdopodobniej przeżywa przygody z Geraltem i spółką :smile:. 52 gry, 8 lat służby i 24 poziom na Steamie.

**Reply by krystian3w** [reply]
Ja mam ledwo 4, a fajne są te ramki:

<div align="center">

https://bit.ly/2YLntKF <img src="https://user-images.githubusercontent.com/35370833/91604963-ef151780-e96f-11ea-8108-9d050846e799.png" alt="presige_frame_anim" width="112" height="112"> https://i.ibb.co/cY2W3HP/presige-frame-anim.png

</div>

**Reply by hawkeye116477** [reply]
A ja mam 2 lata służby, poziom 4 i 100+ gier, ale za kasę kupionych mam tylko tyle, co Krystian ma wszystkich łącznie, a reszta jest z różnych rozdawnictw. Nie wiem czy nawet kiedyś w połowę z nich zagram, ale jak dają promocyjnie za darmo, to trzeba kolekcjonować :smiley:.

Zresztą w przypadku np. wyzwań IG nie podają tytułu gry do klucza, a Steam też nie ma takiej opcji jak Gog, że można podejrzeć co jest przed decyzją, więc nie mam innej opcji jak tylko dodanie. Niby można potem usunąć z półki, ale mogę też kolejny raz wygrać z wyzwania drugi klucz do tej samej gry i znowu wróci, więc to raczej nie ma sensu, wolę już drugi oddać komuś w prezencie.

**Reply by xxcriticxx** [reply]
o boy here we go again

**Reply by hawkeye116477** [reply]
Aha, czyli Critic nie zaprzeczasz, że to twój profil.

A tak mówili, że nie grają, a tu się jednak okazuje, że sporo tutaj graczy, ba nawet weteranów Steama :smile:. 

A Majki też ponoć w CS gra, zapewne goni Ruskich, o ile jeszcze ich nie odcięli od serwerów.

PS: Ja to jestem bardziej konsolowcem, dlatego u mnie taki słaby profil.

**Comment by krystian3w** [comment]
Gwarser usunął (lub mu ktoś przez np. jakiś wyciek poświadczeń/hasła/sesji/być może ostatnia wola z testamentu) jakiś czas temu swoje konto z GitHub (nie ma repozytoriów, posty przeniesiono na @ghost).

![obraz](https://github.com/user-attachments/assets/39e6ee79-a825-4ecf-ad37-61a90c12ab2a) |
--- |

Nie wygląda to na shadowban a nie kojarzę by dodano opcję "uczyń moje konto prywatnym".

**Reply by hawkeye116477** [reply]
Czyli straciliśmy naszego człowieka u gorhilla... 
Nie wiem czy haker by konto usuwał, prędzej by napisał jakieś scamowe wiadomości, a support by wlepił bana.
Może się wyniósł, bo nie spodobał mu się Copilot czy co oni tam ostatnio wymyślili, że słyszałem, że niektórzy programiści rozważali przejście na jakąś inną stronę z tego powodu.

> a nie kojarzę by dodano opcję "uczyń moje konto prywatnym"

W zasadzie tak jakby można, ale tylko to schowa aktywność, a profil raczej zostanie.

Aktualizacja: Sprawdziłem na GitLabie jego profil i twierdzą, że dostał banana. Tak więc wychodzi na to, że albo nagle zaczął rozrabiać albo akcja hakerska.
https://gitlab.com/gwarser

**Reply by krystian3w** [reply]
To z niezablokowanych/nieskasowanych zostaje to:
- na forum Mozilli https://discourse.mozilla.org/u/gwarser (ustawiony na prywatny, być może od założenia tam konta chociaż moderator/admin widzi w Discourse wszystko - ostatni publiczny post z 2023 (wątpię że mają prywatne działy albo hyde park)) i
   -  na bugzilli https://bugzilla.mozilla.org/user_profile?user_id=472632 (~~ostatnie logowanie przed Prima Aprilis/dwa dni przed zniknięciem na GitHub~~ (nowe niby 26 stycznia 2026)).

Być może jeszcze został profil na telepolis, jak przeglądał forum jako zalogowany z tym samym nickiem (chociaż nic na to nie wskazuje lub szybko się odświeżyła pamięć podręczna w Google i nigdy nie był pingowany).

**Reply by krystian3w** [reply]
Ktoś chyba użył jego nicku:

[@gwarser?tab=overview&from=2025-12-01&to=2025-12-31](https://github.com/gwarser?tab=overview&from=2025-12-01&to=2025-12-31)

W API można sprawdzić czy są publiczne komentarze, jak MS nie odciął nas od tego (tu nic nie widzę https://github.com/search?q=commenter%3A%40gwarser&type=issues).

**Comment by krystian3w** [comment]
Z innych nowości lista POL-2 od Olega nie jest już domyślna, uAssets niby zostało ostrzeżone, że będą musieli sami adresować lub od maja zalecać ręczną subskrypcję (chociaż za 4 lata pewnie skończy się tym, że trafi na badlisty jak ktoś nie wyciągnie żywego Olega z pola bitwy):

https://github.com/gorhill/uBlock/commit/d1c57d3fa6de0ae9f2a530ffaccfe0dedec42c60

Niby też towarzystwo nieźle odleciało wymuszając listę CERT (chyba wzorowali się, że nowa ukraińska lista blokuje Pishing/Scam):

https://github.com/gorhill/uBlock/commit/3d2f70ac567f93f40e08fa03bc6df54d1f31bb8a

Anty-adblocków oni nie naprawiają

**Reply by hawkeye116477** [reply]
> Anty-adblocków oni nie naprawiają

Raczej gorhill wie co i jak, bo napisał, że malware.

Ciekawe co by się stało jakby KAD trafił jako domyślny zamiast tego, czy by nagle 1000 zgłoszeń wskoczyło :smiley:.

**Reply by krystian3w** [reply]
CERT wywalili za za dużo hostów:

https://github.com/gorhill/uBlock/commit/e713e133eb16f1d1cc9b50044cecbcaa7d9ba65f

> użytych 547 185 z 547 190

Nie podano limitu ile to maksimum i czy konwersja w regex lub dzikie karty wchodzi w grę.

**Comment by hawkeye116477** [comment]
Albo GH shackowali miłośnicy pankejków albo ktoś w MS sobie jaja robi :smile:
<img width="879" height="102" alt="2026-07-30_191208" src="https://github.com/user-attachments/assets/2475a3da-3f6d-4384-b201-f4f2286e279a" />
